test/e2e/node/security_context.go

```diff
@@ -269,7 +270,10 @@ func testPodSELinuxLabeling(f *framework.Framework, hostIPC bool, hostPID bool)
 	err = f.WaitForPodRunning(pod.Name)
 	framework.ExpectNoError(err, "Error waiting for pod to run %v", pod)
 
-	content, err = tk.ReadFileViaContainer(pod.Name, "test-container", testFilePath)
-	framework.ExpectNoError(err, "Error reading file via container")
-	gomega.Expect(content).NotTo(gomega.ContainSubstring(testContent))
+	// for this to work, SELinux should be in enforcing mode, so let's check that
+	isEnforced, err := tk.ReadFileViaContainer(pod.Name, "test-container", "/sys/fs/selinux/enforce")
```

Comment:
I guess `err != nil` case is intentionally ignored here.
It is necessary to add comment which explains why here to avoid adding `err != nil` check in the future.

Reply:
Yes, if we get error reading this file, we assume that selinux is not in enforced mode, and skip the test.

Perhaps we could use something like this instead?
```go
isEnforced, _ := tk.ReadFile...
if isEnforced == "1" {
...
```

@oomichi Let me know if that's OK with you.

Reply:
I guess the best way is `No such file or directory` error is handled as ignored and the other `err` leads an exception.

Reply:
Not sure what you mean by exception -- `f,Failf("can't read file: %v", err)`?

In both cases, I guess, we have to skip the test (rather than fail it), which is what the current code does.

I'll add error reporting

Reply:
Oh well, the error is actually already logged by `tk.ReadFileViaContainer()` itself, so I'm not sure what else to do here.

Reply:
except maybe what I suggested [above](https://github.com/kubernetes/kubernetes/pull/88247#discussion_r381026024), i.e. ignoring the error in a more straightforward way